pyspline/pyCurve.py

```diff
@@ -802,15 +802,15 @@ def projectCurveMultiSol(self, inCurve, nIter=25, eps=1e-10, **kwargs):
 
         return np.array(uSol), np.array(tSol), np.array(diff)
 
-    def computeData(self):
+    def computeData(self, recompute=False):
```

Comment:
Can you add a doc-string for the recompute flag? (this applies to the other `computeData` functions

Reply:
yep